packages/node/src/plugins/segmentio/context-batch.ts

```diff
@@ -19,9 +30,7 @@ export class ContextBatch {
   constructor(maxEventCount: number) {
     this.maxEventCount = Math.max(1, maxEventCount)
   }
-  public tryAdd(
-    item: PendingItem
-  ): { success: true } | { success: false; message: string } {
+  public tryAdd(item: PendingItem): ContextBatchResult {
```

Comment:
Confused about why this would be neccessary =S.

Reply:
Related to above, I thought defining the return types would be useful, but we can resolve the error:

```
node_modules/@segment/analytics-node/src/plugins/segmentio/publisher.ts(178,39): error TS2339: Property 'message' does not exist on type '{ success: true; } | { success: false; message: string; }'.
  Property 'message' does not exist on type '{ success: true; }'.
```

By using:

```
      if ('message' in fbAddStatus) {
        ctx.setFailedDelivery({
          reason: new Error(fbAddStatus.message),
        })
      }
```